lib/phx_tools_web/live/phx_tools_live/instruction_components.ex

```diff
@@ -0,0 +1,105 @@
+defmodule PhxToolsWeb.InstructionComponents do
+  @moduledoc """
+  Shows the steps to execute different scripts depending on the operating system.
+  """
+  use PhxToolsWeb, :html
+
+  @spec linux(map()) :: Phoenix.LiveView.Rendered.t()
+  def linux(assigns) do
```

Comment:
Why 2 components? Most of the code is duplicated.
Use [slots](https://hexdocs.pm/phoenix_live_view/Phoenix.Component.html#module-slots) instead.